### ISSUE TITLE
Redirected to wrong url when session have expired

### ISSUE THREAD

**Issue by AdmiringWorm** [chocolatey/chocolatey.org#522]
### What You Are Seeing?
When the current session have expired and you're trying to navigate to a location that requires you to be authenticated, you get redirected to `/Account/Login` which doesn't exist and a 404 page is shown instead.

### What is Expected?
Expected to be redirected to `/users/account/LogOn` instead.

### How Did You Get This To Happen? (Steps to Reproduce)
1. Log in to chocolatey.org
2. Wait for the session to expire (don't know how long that takes)
3. Try to navigate to the Account page (using the link)
4. Behold the 404 page.

/cc @flcdrg

**Comment by ferventcoder** [comment]
It's the right url, however there is an issue that it doesn't grab the right route due to the issue with timeouts.

**Comment by AdmiringWorm** [comment]
I see, I thought the correct link was `/users/account/LogOn` since that's what the Log in link is using.

**Comment by ferventcoder** [comment]
Yes, they both are - the link failing is the built-in link that would redirect to the latter.

**Comment by ferventcoder** [comment]
Looks like something is being ignored. Have a few things to add for this.

**Comment by ferventcoder** [comment]
Adding the AAR here:

* 20170603 18:04 CDT: Deployed documentation updates to the website - last deployment prior was 20170526 at 15:38 CDT.
* 20170603 22:51 CDT: 
  First known question at https://devopslibrary.slack.com/archives/C114J9G68/p1496548293298647 (link has login) -
    ~~~
    glenn.sarti [22:51] 
    Is the chocolatey login page working? Keep getting timeouts in IE, Edge and Chrome  
    @ferventcoder
    ~~~
* 20170604 04:15 CDT: @flcdrg Noted issues with login pointing to wrong location and @AdmiringWorm confirmed - [June 4, 2017 4:15 AM](https://gitter.im/chocolatey/chocolatey.org?at=5933cf9d7503e2b7060c3dea)
* 20170604 06:03 CDT: Confirmed by @ferventcoder - [June 4, 2017 6:03 AM](https://gitter.im/chocolatey/chocolatey.org?at=5933e8ed167d046a7d68a587)
* 20170604 06:38 CDT: First notification of issue to bigger audience by @ferventcoder - [June 4, 2017 6:38 AM](https://gitter.im/chocolatey/chocolatey.org?at=5933f14acf9c13503c50f006)
* 20170604 11:07 CDT: After noting this wasn't fixed with regular maintenance and would need closer investigation, @ferventcoder sent wider notice at https://twitter.com/chocolateynuget/status/871397640667250688 and https://groups.google.com/forum/#!topic/chocolatey/DiCKItlTOMs
* 20170604 11:27 CDT: @ferventcoder provides progress so far - [June 4, 2017 11:27 AM](https://gitter.im/chocolatey/chocolatey.org?at=5934350a6549436c7d160e0f) 
    * Status so far - database maint to clean up indexes and stats
    * Worked with appharbor to see if there were any issues in connectivity from the machine
    * Increased memory allowances with memcached as it was hitting in 95+ percentile of allowance - now we are down to 50% usage of plan limits
    * Deployed status message to the site
    * Investigating reducing size of items being cached by separating package files (from package queries) to get a better hit ratio on memcached
    * As the issue just cropped up without any recent code changes we believe it is the size of the data being too large to cache - the Chocolatey package itself being the main culprit
    * Active TCP connections have soared to over 2K, normally they sit at 300 or less
No significant signs of a DDoS attack
* 20170604 12:23 CDT: @ferventcoder deployed [#474](https://github.com/chocolatey/chocolatey.org/issues/474) to split up and reduce cache sizes to get a better ratio of storage on memcached [June 4, 2017 12:23 PM](https://gitter.im/chocolatey/chocolatey.org?at=5934422231f589c64f73c986)
* FIX: 20170605 13:27 CDT: @ferventcoder pushes #522 to production (see [ef14876fe](https://github.com/chocolatey/chocolatey.org/commit/ef14876fe5582aa83663a1c60711aed888642397) for details) - [June 5, 2017 1:27 PM](https://gitter.im/chocolatey/chocolatey.org?at=5935a2adf31c8ced0c070df7) 
* 20170605 13:40 CDT: Notices are sent when it is determined this fixes the issue https://twitter.com/chocolateynuget/status/871798925396594688 and https://groups.google.com/forum/#!topic/chocolatey/DiCKItlTOMs

The setting changes for the web.config immediately fix the issues. The weirdness is that these settings appear to have been needed all along for the last few years (known issues with ASP.NET MVC), but only cropped up after the deployment on June 3rd. The only known recent change was an upgrade of the hosting servers to .NET 4.7. Still investigating with AppHarbor on what could have caused this and will provide details once we determine the issue (if it is not the update to .NET 4.7).
